spec/support/quota_helper.rb

```diff
@@ -117,14 +122,32 @@ def google_model
     @vm_template = FactoryGirl.create(:template_google, :ext_management_system => ems)
     m2_small_flavor = FactoryGirl.create(:flavor_google, :ems_id => ems.id, :cloud_subnet_required => false,
                                          :cpus => 4, :cpu_cores => 1, :memory => 1024)
-    create_request(:number_of_vms => 1, :owner_email => 'tester@miq.com',
-                                         :src_vm_id      => @vm_template.id,
-                                         :boot_disk_size => ["10.GB", "10 GB"],
-                                         :placement_auto => [true, 1],
-                                         :instance_type  => [m2_small_flavor.id, m2_small_flavor.name])
+    create_request(:number_of_vms => 1, :owner_email    => 'tester@miq.com',
```

Comment:
@billfitzgerald0120 
for the owner email we should typically use user@example.com.

Not important but for future reference